### PR TITLE
Update styles of StepperInput in price calculators

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

- Style `StepperInput` to match Figma
- Pass label from configs, we did not have it
- Update translations for `numberCoInsured` to match Figma

<!-

What changes are made?
If there are many changes, a list might be a good format.
If it makes sense, add screenshots and/or screen recordings here.
-->

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4536?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4536** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4536?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4535** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4535?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4534** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4534?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4531** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4531?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4530** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4530?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4528** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4528?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4527** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4527?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4526** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4526?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4524** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4524?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4522** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4522?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4521** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4521?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4520** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4520?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4519** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4519?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4518** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4518?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4517** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4517?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4516** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4516?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4513** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4513?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4510** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4510?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4508** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4508?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4507** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4507?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4506** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4506?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/public/locales/en/bankid.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
```
Comment (guilhermespopolin) [inline apps/store/public/locales/en/bankid.json:1]: You can ignore those as they're a part of an PR that I've opened.

File: apps/store/src/components/StepperInput/StepperInput.css.ts [hunk 2]
```diff
@@ -3,11 +3,10 @@ import { tokens } from 'ui'
 
 export const outerWrapper = style({
   display: 'flex',
-  justifyContent: 'space-between',
-  alignItems: 'center',
-  gap: tokens.space.md,
   width: '100%',
-  minHeight: '3rem',
+  height: '4rem',
+  boxSizing: 'border-box',
```
Comment (guilhermespopolin) [inline apps/store/src/components/StepperInput/StepperInput.css.ts:8]: I thought this was being applied as part of the CSS reset, no? 🤔

**Review by guilhermespopolin — APPROVED** [review]
Well done. Just left a minor comment.

File: apps/store/src/components/StepperInput/StepperInput.css.ts (hunk 2 above)
Comment (alebedev) [reply apps/store/src/components/StepperInput/StepperInput.css.ts:8]: Maybe it was like this some time ago, but this does not work right now

Comment (alebedev) [reply apps/store/src/components/StepperInput/StepperInput.css.ts:8]: We have a rule in global styles, will investigate why it does not apply

**Comment by alebedev** [comment]
### Merge activity

* **Jul 25, 7:42 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4536) rebased this pull request as part of a merge.

**Pull request merged**